PySpice/Spice/HSpice/RawFile.py

```
@@ -0,0 +1,122 @@
+            nodes = [WaveForm.from_unit_values(name, u_V(np.array([val]))) for name, val in self.op_nodes.items()]
+            branches = [WaveForm.from_unit_values(name, u_A(np.array([val]))) for name, val in self.op_branches.items()]
+            return OperatingPoint(
+                simulation=self.simulation,
+                nodes=nodes,
+                branches=branches
+            )
+
+        # data is a list of sweeps returned by hspice_read
+        # data[0] is sweeps tuple (sweep, sweepValues, dataList)
+        sweeps = self.data[0][0]
+        scale_name_outer, sweep_values, data_list = sweeps
```

Ruff (0.15.18)

[warning] Unpacked variable `scale_name_outer` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)